src/AgGrid.fs

```diff
@@ -762,6 +781,10 @@ type AgGrid<'row> =
             static member inline aggFunc(v: AggregateFunction) = columnDefProp<'row, 'value> ("aggFunc" ==> v.AggregateText)
             static member inline rowGroup(v: bool) = columnDefProp<'row, 'value> ("rowGroup" ==> v)
 
+            static member inline suppressAggFuncInHeader(v:bool) = columnDefProp<'row, 'value> ("suppressAggFuncInHeader" ==> v)
+            static member inline GroupCellRendererParams(v: IGroupCellRendererParams<'row, 'value>) =
```

Comment:
```suggestion
            static member inline cellRendererParams(v: IGroupCellRendererParams<'row, 'value>) =
```

I'd keep the method name the same as what AG Grid calls it, to minimise the amount of translation a developer has to make from the AG Grid docs to the F# code that they have to write.

Reply:
I agree in general, but this is an overload of the regular cellRendererParams, with a few settings that are only available for Group columns

Reply:
Discussed offline.